controller.py

```diff
@@ -229,6 +229,15 @@ def search(self, location, live=True):
             return Response(body, 200, headers)
 
     def libraries(self):
+        # Return a specific set of information about all libraries in production; this generates the library list in the admin interface,
+        libraries = []
+        in_production = self._db.query(Library).filter(Library.registry_stage=="production").order_by(Library.name)
+        for library in in_production:
+            uuid = library.internal_urn.split("uuid:")[1]
+            libraries += [self.library_details(uuid, library)]
+        return dict(libraries=libraries)
+
+    def libraries_qa(self):
```

Comment:
Elsewhere in this web app we define a single controller method that takes a `live` argument which toggles whether to look at all libraries or only production libraries. We add two routes to the same method with different arguments. Take a look at `nearby` and `nearby_qa` in `app.py` for an example.

This guarantees that we always build the library list using the same code -- only the query that chooses libraries is different.